node/service/chain-specs/example.json

```diff
@@ -95,6 +95,36 @@
           ]
         ]
       },
+      "democracy": {
+        "phantom": null
+      },
+      "council": {
+        "phantom": null,
+        "members": []
+      },
+      "technicalCommittee": {
```

Comment:
The `technicalCommittee` pallet was removed when we introduced `OpenGov`, you should remove it from the `example.json` file. We have `techComm` now.